backend/services/engine_evidence.py

```
@@ -0,0 +1,116 @@
+) -> dict[str, Any]:
+    """Return fixed-shape evidence; actual fields stay null until an instance loads."""
+    isolated = bool(
+        getattr(engine_cls, "_is_subprocess_isolated", False)
+        or getattr(engine_cls, "runs_out_of_process", False)
+    )
+    loaded = False
+    if instance is not None:
+        contract = getattr(instance, "execution_evidence_loaded", False)
+        try:
+            loaded = bool(contract() if callable(contract) else contract)
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)